### PR TITLE
Fix Web and Dotnet issue 477

### PR DESCRIPTION
**Pull request by mijin85cho**
Signed-off-by: Mijin, Cho <mijin85.cho@samsung.com>

### Change Description ###

Fix Web and Dotnet issues reported on #477.

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/526

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/526

File: docs/application/web/guides/data/file-archiving.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ To start any kind of ZIP operation (packing or unpacking), you must first call t
 **Table: File modes**
```
Comment (mijin80lee) [inline docs/application/web/guides/data/file-archiving.md]: line31,32,33,34: Remove the blank in front of <br> in the table.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/526

**Comment by mijin85cho** [comment]
Thanks for the review.

**Comment by mijin80lee** [comment]
TD Sync Completed.(cross-check)

**Pull request merged**